### ISSUE TITLE
Off-by-one error in mypng.rotate(90) vs mypng.transpose(Image.ROTATE_90)?

### ISSUE THREAD

**Issue by jordanapplewhite** [python-pillow/Pillow#2069]
### What did you do?
I rotated a bunch of .pngs like so:
```
    i = Image.open(path + "/" + file)
    i = i.rotate(270)
    i.save(path + "/" + file)
```

### What did you expect to happen?
I expected the bitmap data to be unchanged aside from the rotation. 

### What actually happened?
If Expand=true, the bitmap dimensions grow by one on each axis. A 16x20 bitmap becomes 21x17.
If Expand=false, the dimensions stay as expected but the image is shrunk by a row on each axis.  

### How I fixed this issue
The bitmap data is preserved by using the transpose method instead of rotate:
 ```
    i = Image.open(path + "/" + file)
    i = i.transpose(Image.ROTATE_270)
    i.save(path + "/" + file)
```

### What versions of Pillow and Python are you using?
python 2.7.11
pillow 3.2.0

I read somewhere that there is no difference between i.rotate(270) and i.transpose(Image.ROTATE_270) but I am experiencing different behavior between the two so I thought I'd let you know. Please let me know if i can provide more info. Thanks!

**Comment by radarhere** [comment]
Hi. I'm unable to reproduce the problem. If you could provide one of the image files that you experienced this with, that would be helpful.

**Comment by wiredfool** [comment]
I think that there are recent changes that fix this, in either 3.3 or master.

**Comment by jordanapplewhite** [comment]
Yup, upgrading from 3.2 to 3.3 fixed this behavior. Thanks for the tip!
